cmd/internal/shared/console/config.go

```diff
@@ -41,7 +41,7 @@ var DefaultConsoleConfig = console.Config{
 			AssetsBaseURL: shared.DefaultAssetsBaseURL,
 			IconPrefix:    "console-",
 			CSSFiles:      []string{"console.css"},
-			JSFiles:       []string{"console.js"},
+			JSFiles:       []string{"libs.bundle.js", "console.js"},
```

Comment:
This may need a mention in CHANGELOG, because deployments that customized this (I can't imagine there are any, but still) would need to update their config.